lib/install.js

```diff
@@ -397,13 +409,15 @@ function install(opts, cb) {
     var yauzl = require('yauzl');
     var extractPath = path.join(path.dirname(zipFilePath), path.basename(zipFilePath, '.zip'));
 
-    yauzl.open(zipFilePath, {lazyEntries: true}, function onOpenZipFile(err, zipFile) {
```

Comment:
Can you comment on why this was removed and what's the impact on other drivers?

Reply:
@vvo the previous code took the first file it found inside the zip and just unzipped it in the destination folder. This worked fine for all other webdrivers as the downloaded zip for those contained nothing else, but the webdriver binary.
Unfortunately, for the chromium edge driver this is not the case, as the zip has this structure:
```
Driver_Notes
-- | credits.html
-- | LICENSE
msedgedriver.exe
```
The previous code, because it only unzipped the first file it found, kept unzipping the `Driver_Notes/credits.html` file and it `selenium-standalone start` obviously failed as an html file can't be used as a webdriver.

The changes have no effect for other webdrivers, what the new code does is ignore all folders inside the zip, and unzip the first file inside it (so the other webdrivers' zip structure works as well)

Reply:
👍